Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -471,7 +471,7 @@ public static InputActionAsset FromJson(string json)
         /// in the asset.
         /// </summary>
         /// <param name="actionNameOrId">Name of the action as either a "map/action" combination (e.g. "gameplay/fire") or
-        /// a simple name. In the former case, the name is split at the '/' slash and the first part is used to find
+        /// a simple name e.g. "fire". In the former case, the name is split at the '/' slash and the first part is used to find
```

Comment:
This would better read as (e.g. "fire") to be uniform with the above sentence, or otherwise write in full: 'a simple name such as "fire" for example').

Reply:
Thanks for the suggestions, accepted it so will resolve this one since outdated.